README.md

```diff
@@ -11,3 +11,8 @@ You can see the content in different ways locally:
 - Start the Hugo server with drafts enabled: `hugo server -D`
 - Build Hugo website  in `/public`: `hugo`
 - Build Hugo website with drafts enabled in `/public`: `hugo -D`
+
+## Hugo Versions
+
+Until we find a solution, it is recommended to use version 0.79.0 of Hugo.
+For convenience, you can find the Linux binary under [binaries](./binaries/) folder.
```

Comment:
It is somewhat difficult to know which version the binary is.
Does it still make sense to add the tar.gz of if with the proper name? 

Note: I do not have a strong preference either...

Reply:
I thought of this too but I think we should not keep both files for sure, adds unnecessary overhead. The version is documented in readme so it is fine to remove the tar ball.